sdk/loadtesting/azure-developer-loadtesting/swagger/README.md

```diff
@@ -5,14 +5,21 @@ require: https://github.com/Azure/azure-rest-api-specs/blob/3e27c70e7c02c07b458b
 java: true
 data-plane: true
 title: LoadTestingClient
-package-version: 1.0.0-beta.2
+package-version: 1.0.0
 security: AADToken
 security-scopes: https://cnt-prod.loadtesting.azure.com/.default
 artifact-id: azure-developer-loadtesting
 namespace: com.azure.developer.loadtesting
-generate-builder-per-client: false
+generate-builder-per-client: true
 partial-update: true
 output-folder: $(azure-sdk-for-java-folder)/sdk/loadtesting/azure-developer-loadtesting
 service-versions:
 - '2022-11-01'
+directive:
+- rename-operation:
+    from: LoadTestRun_ListMetricNamespaces
+    to: LoadTestRun_GetMetricNamespaces
+- rename-operation:
+    from: LoadTestRun_ListMetricDefinitions
+    to: LoadTestRun_GetMetricDefinitions
```

Comment:
Can this change me made directly in the swagger?

Reply:
Yes, we are currently discussing changing in swagger. However, it might take sometime as it is a breaking now since the docs have been published. For the time being, can this be handled using directives?